utils/s3/client.cc

```diff
@@ -198,9 +198,49 @@ future<semaphore_units<>> client::claim_memory(size_t size) {
     return get_units(_memory, size);
 }
 
+[[noreturn]] void map_s3_client_exception(std::exception_ptr ex) {
```

Comment:
You moved this function out of this file by previous patch. Either don't move it back by this patch, or don't move it by that (previous) one